### ISSUE TITLE
qTox doesn't use the standard data location on Windows

### ISSUE THREAD

**Issue by tsdko** [qTox/qTox#585]
qTox saves its data files to `%LOCALAPPDATA%\Tox\qTox` instead of the standard `%APPDATA%\Tox`.

**Comment by tux3** [comment]
I'm a bit conflicted on this. If we put the data file in %APPDATA%, then it becomes Roaming data which is supposed to be kept on a remote server and/or cloud, tied to the account instead of the local machine. That includes your Tox private key and chat history.

But %LOCALAPPDATA% is semantically wrong, this isn't actually device-specific data or too large to roam, so I guess we should move it to %APPDATA%. The user should probably be able to keep his qTox settings synced with his account, instead of stuck on the local device.

As for replacing "Tox\qTox\" with just "Tox\", iirc the format has always been "Company\Product\" so this should be correct. It also avoids conflicting with other clients by mistake.

TL;DR: Is anyone concerned that their Tox data (private key, chat history, settings, ...) could end up on a remote server and/or msft's cloud, or is it okay to make that change right now ?

**Comment by tsdko** [comment]
> As for replacing "Tox\qTox\" with just "Tox\", iirc the format has always been "Company\Product\" so this should be correct. It also avoids conflicting with other clients by mistake.

Well, I mostly opened this issue because qTox couldn't find the tox_save file from an other client, wouldn't it be possible to at least keep the profiles in %APPDATA%\Tox for compatibility with other clients?

**Comment by tux3** [comment]
Ah you're right, this is actually part of the Single Tox Standard.

I'll make the change right now.
